infra/discovery/src/listener/handler_marketplace.js

```diff
@@ -32,14 +37,6 @@ function isOfferEvent(eventName) {
   return OFFER_EVENTS.includes(eventName)
 }
 
-function getOriginListingId(networkId, event) {
```

Comment:
@franckc @mikeshultz I had a need to reuse these two functions in the DApp. I was not sure about adding `@origin/discovery` as a dependency for the DApp. So I moved these two functions to `@origin/graphql` package since both the DApp and `@origin/discovery` are dependent on that already.

Let me know if you have any other suggestions for this.

Reply:
I have no issues with moving it and nothing else comes to mind for a better place.  Is the `esmImport` necessary?

Reply:
I don't think we can `require` it, since `@origin/graphql` package uses ES6 modules (`import`/`export`). We don't have babel in `@origin/discovery` package. Could have added that but every other file uses `esmImport`. So used that for consistency.